src/main/java/org/mule/extension/email/internal/mailbox/imap/IMAPOperations.java

```diff
@@ -233,5 +234,36 @@ public void moveToFolder(@Connection MailboxConnection connection,
     }
   }
 
+  /**
+   * Counts the emails in the {@code mailboxFolder}.
+   * <p>
+   *
+   * @param connection          The corresponding {@link MailboxConnection} instance.
+   * @param mailboxFolder       Mailbox folder where the emails are.
+   */
+  @Summary("Get the total amount of messages in a specified mailbox folder")
+  @DisplayName("Count messages - IMAP")
+  @Throws(EmailMarkingErrorTypeProvider.class)
+  public int countMessagesImap(@Connection MailboxConnection connection,
```

Comment:
doubt is reasonable.
I think the IMAPOperations only accept IMAP connections. At least I don't see the option in studio to do it otherwise. 

<img width="908" alt="Screen Shot 2022-05-20 at 10 03 54" src="https://user-images.githubusercontent.com/8380017/169534013-1b47ce0b-d1ac-4da0-99ec-58a434f1421c.png">
<img width="612" alt="Screen Shot 2022-05-20 at 10 04 04" src="https://user-images.githubusercontent.com/8380017/169534069-54136672-782f-4741-a0b9-2c7757e7785d.png">